projects/batfish/src/main/java/org/batfish/main/Batfish.java

```diff
@@ -3169,6 +3178,97 @@ private void populateFlowHistory(FlowHistory flowHistory, String environmentName
     }
   }
 
+  // this is a hack to deal with router names that are named ethX
```

Comment:
I don't understand this comment. What does any of this code have to do with the name of the router? Did you mean interface names? If so, you should just use normalized name of the interface in the host config to be merged. Alternatively, use the appropriate normalization function for the vendor of the base configuration to be merged into.